tests/session_metadata_api_test.py

```diff
@@ -17,3 +17,12 @@ def setUpClass(cls):
 
     def filter_app_session_metadata_test(self):
         self.assertEqual(200, self.sdk.filter_app_session_metadata(self.org_uuid, self.app_id).status_code)
+
```

Comment:
Typically, test methods should start with `test_` to ensure that the testing framework (like `unittest` or `pytest`) recognizes them as tests.

If self.org_uuid, self.app_id, and self.session_id are being initialized in multiple test methods, consider using a setUp method to initialize them once. This keeps your test methods clean and DRY (Don't Repeat Yourself).

`def setUp(self):
    self.org_uuid = "your_org_uuid"
    self.app_id = "your_app_id"
    self.session_id = "your_session_id"`